trade-module/src/main/java/io/edpn/backend/trade/application/dto/FindCommodityResponse.java

```diff
@@ -0,0 +1,35 @@
+package io.edpn.backend.trade.application.dto;
+
+
+import lombok.Builder;
+import lombok.Value;
+
+import java.time.LocalDateTime;
+
+@Value
+@Builder
+public class FindCommodityResponse {
+    LocalDateTime pricesUpdate;
+    String commodityName;
+    Station station;
+    System system;
+    
+    @Value
+    @Builder
+    public static class Station {
+        String name;
+        Double arrivalDistance;
+        String landingPadSize;
+        Boolean planetary;
+        Boolean requireOdyssey;
+        Boolean fleetCarrier;
+    }
+    
+    @Value
+    @Builder
+    public static class System {
+        String name;
+        CoordinateDTO coordinateDTO;
```

Comment:
apply the same CoordinateDTO to `CommodityMarketInfoResponse` for consistency
I also suggest renaming to coordinates

```suggestion
        CoordinateDTO coordinates;
```